### ISSUE TITLE
Limit project title per organization

### ISSUE THREAD

**Issue by SteinRobert** [unikubehq/projects#7]
Currently it is possible to create multiple project's with the same name per organisation.
This can be very confusing, especially in context of the CLI (https://github.com/unikubehq/cli/).

I propose a validation to permit duplicated project names per organisation.

What do you guys think @Schille @buschNT @schwobaseggl ?

**Comment by buschNT** [comment]
> I propose a validation to permit duplicated project names per organisation.

I agree. Furthermore its required for the CLI to solve the issue: https://github.com/unikubehq/cli/issues/20
Actually, I thought that it was already decided that we want to permit duplicate project names per organization.
